### ISSUE TITLE
blurry text

### ISSUE THREAD

**Issue by harridu** [docker/docker.github.io#5904]
File: [index.md](https://docs.docker.com/), CC @johndmulhausen 
Hi folks, 
the text on docs.docker.com appears to be blurry and difficult to read. See the attached enlarged snapshot showing what I mean. Do you see that some parts of the chars are white, others are light blue, etc. Horizontal lines appear to be "more white" than vertical lines.

I tried both Google Chrome and Firefox on Linux, as well as Safari on my Mac. Same problem. 

The second snapshot (snap_emacs) shows the same text, copied-and-pasted into an editor running on the same display, just to show the difference. The font is smaller and pixeled, but its way easier to read. IMHO.

Thanx for your support
Harri
![snap](https://user-images.githubusercontent.com/832164/35735537-7d6689ee-0825-11e8-8998-30fd0383426d.gif)

![snap_emacs](https://user-images.githubusercontent.com/832164/35735548-84e7c7fa-0825-11e8-9f12-14bf0a3a8fb2.gif)

**Comment by mistyhacks** [comment]
@jsouth PTAL? It's not really a fair comparison because the second example is using a fixed-width font and is white on black, and the top example is using light grey on dark blue (looks like) and a variable-width font. 

@harridu do you see the same problem when you are not using night mode?

**Comment by jsouth** [comment]
@mistyhacks / @harridu 

OK - will investigate further / test across platforms and circle back 👍

In safari on mac : I see:
http://recordit.co/kBW180YFxc

**Comment by harridu** [comment]
Yes, I think its even worse for light background. See the new snapshot attached here.

Wrt "fair comparison": The actual colors and fonts used are probably defined in a CSS file, i.e. beyond my control.
![snap_white](https://user-images.githubusercontent.com/832164/35765708-fc926194-08c9-11e8-912c-5e08929ab471.gif)

**Comment by L-Hudson** [comment]
Closing this ticket due to its age, and the impending refactor. If you think this is in error, feel free to reopen. Thanks!
